### PR TITLE
[IMP] CRM: update multiple sales teams doc

### PR DESCRIPTION
**Pull request by vpk-odoo**
Associated task: https://www.odoo.com/web#id=2588786&action=333&active_id=3835&model=project.task&view_type=form&cids=3&menu_id=4720

### CODE REVIEW

File: content/applications/sales/crm/pipeline/multi_sales_team.rst [hunk 1]
```diff
@@ -1,47 +1,78 @@
...
-    :align: center
+.. image:: multiple_sales_teams/sales-team-overview.png
+   :align: center
+   :alt: Sales Team Overview
 
-If you now filter on this specific channel in your pipeline, you will
-find all of its opportunities.
+Clicking on the three little dots opens another menu where you can quickly access information,
+such as viewing quotations, creating a new opportunity, or opening a report. Additionally, you can
+choose a color for your team as well as access the team configuration template.
```
Comment (meng-odoo) [inline content/applications/sales/crm/pipeline/multi_sales_team.rst:69]: Editing this entire paragraph for brevity.

`Clicking on the three dots in the corner of a tile opens a navigational menu that lets you quickly view documents or reports, create new quotations or opportunities, pick a color for this team, or access its configuration page.`

File: content/applications/sales/crm/pipeline/multi_sales_team.rst [hunk 2]
```diff
@@ -1,47 +1,78 @@
...
-.. image:: media/multi_sales_team03.png
-    :align: center
+.. image:: multiple_sales_teams/team-overview-three-dot-menu.png
+   :align: center
+   :alt: Three Dot Menu
 
-Sales channel dashboard
-=======================
+If you are a sales manager managing multiple teams, you can easily tap on *Pipeline*, and Odoo will
+take you directly to that team’s pipeline without having to access the Sales menu.
```
Comment (meng-odoo) [inline content/applications/sales/crm/pipeline/multi_sales_team.rst:76]: Editing this entire paragraph for brevity and accuracy.

`Clicking on the **Pipeline** button brings you directly to that team's CRM pipeline.`

**Review by meng-odoo — CHANGES_REQUESTED** [review]
@StraubCreative Hi Zach! I made tons of changes to VPK's content on this doc, especially since you suggested I edit to make it shorter/clearer/more concise. Please let me know if you have any questions/comments about my changes below :) Thanks

File: content/applications/sales/crm/pipeline/multi_sales_team.rst [hunk 3]
```diff
@@ -1,47 +1,78 @@
...
+   A team name, who you accept emails from, and a minimum score are all required form fields.
+
+Add members to your Sales Team
+------------------------------
+
+Team members can be added directly to the Sales Teams creation template when creating a new team or
+at a later time via editing. Click on *Add* under the *Assignment* tab, and choose a salesperson by
+typing in their name, selecting them from the drop down, or adding in a new name. If your team wants
+to ensure that your salespeople are not overdoing it, you can set a maximum number of leads in a 30
+day period.
```
[the remark this answers is not in the document]
Comment (StraubCreative) [reply content/applications/sales/crm/pipeline/multi_sales_team.rst:37]: Yea reformatting entire block is fine. Made small grammar change from `30 day period` to `30-day period` (with hyphen).

**Comment by StraubCreative** [comment]
Hi @meng-odoo 

Good first pass 👍  
I made all of the change requests you asked for which you can see in commit [8824706](https://github.com/odoo/documentation/commit/88247063f8867c403468aab6969df185a5937ae4)

For next round of review, I highly suggest reworking the tense and tone for the whole document, namely:
- there are a number of instances where verb tense is in _continuous_ present, where I think it would be more ideal to write in the _simple_ present instead (e.g. lines 56 and line 64, _clicking_ vs. _click_). While continuous present is _technically_ correct, I’d argue that this tense makes for more clunky sentence structures that don’t clearly communicate direct paths of action ([guidelines reference on tense]( https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#grammatical-tenses))
- there’s a lot of 2nd person familiar pronouns in this doc (e.g. you, your)…can we clean these up a bit so it sounds less accusatory and more instructional? We’ll especially want to remove these in headings per [guidelines on headings](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#titles-and-headings).

Looking forward to your thoughts and further improvements as you see fit!

File: content/applications/sales/crm/pipeline/multi_sales_team.rst [hunk 4]
```diff
@@ -1,47 +1,66 @@
...
-    :align: center
+On the creation page, you can set an *Email Alias* for your team. Every message sent to that email
+address will create a lead/opportunity in your pipeline. You can also choose whether you
+accept emails from *Everyone*, *Authenticated Partners*, or *Followers Only*.
 
-Add members to your sales channel
----------------------------------
+You can set up a *Domain* if you want to assign leads/opportunities to your team based on specific
+filters, such as country, language, etc. Set an *Invoicing Target* if your team has specific
+monthly revenue targets they need to achieve.
```
Comment (meng-odoo) [inline content/applications/sales/crm/pipeline/multi_sales_team.rst:21]: Removing "you" and improving wording

`Set a *Domain* to assign leads/opportunities to this Sales Team based on specific filters, such as country, language, or campaign. Set an *Invoicing Target* if this team has specific monthly revenue goals.`

File: content/applications/sales/crm/pipeline/multi_sales_team.rst [hunk 5]
```diff
@@ -1,47 +1,66 @@
...
-You can add members to any channel; that way those members will see the
-pipeline structure of the sales channel when opening it. Any
-lead/opportunity assigned to them will link to the sales channel.
-Therefore, you can only be a member of one channel.
+.. image:: multiple_sales_teams/sales-team-creation.png
+   :align: center
+   :alt: Creating a Sales Team
 
-This will ease the process review of the team manager.
+Add members to your Sales Team
```
Comment (meng-odoo) [inline content/applications/sales/crm/pipeline/multi_sales_team.rst:27]: Removing "you"

`Add members to a Sales Team`

File: content/applications/sales/crm/pipeline/multi_sales_team.rst [hunk 6]
```diff
@@ -1,47 +1,66 @@
...
+------------------------------
 
-.. image:: media/multi_sales_team02.png
-    :align: center
+You can add team members to a Sales Team by clicking **Add** under the Assignment tab when editing
+the Sales Team's configuration page. Choose a salesperson by selecting them from the dropdown menu
+or creating a new salesperson. To ensure that your salespeople are not overworking, you can set a
+maximum number of leads that they can be assigned in a 30-day period.
```
Comment (meng-odoo) [inline content/applications/sales/crm/pipeline/multi_sales_team.rst:33]: Removing "you" and improving wording/tense

`To add team members, click **Add** under the Assignment tab when editing the Sales Team's configuration page. Select a salesperson from the dropdown menu or create new salesperson. Set a maximum number of leads that can be assigned to this salesperson in a 30-day period to ensure that they do not overwork.`

File: content/applications/sales/crm/pipeline/multi_sales_team.rst [hunk 7]
```diff
@@ -1,47 +1,66 @@
...
-If you now filter on this specific channel in your pipeline, you will
-find all of its opportunities.
+.. image:: multiple_sales_teams/add-a-salesperson.png
+   :align: center
+   :alt: Adding a Salesperson
 
-.. image:: media/multi_sales_team03.png
-    :align: center
+You can add someone as a team member or a Team Leader to multiple Sales Teams, allowing them to
+easily check all of the pipelines they need access to.
```
Comment (meng-odoo) [inline content/applications/sales/crm/pipeline/multi_sales_team.rst:40]: Removing "you" and improving wording

`One person can be added as a team member or Team Leader to multiple Sales Teams, allowing them to
access all of the pipelines that they need to.`

File: content/applications/sales/crm/pipeline/multi_sales_team.rst [hunk 8]
```diff
@@ -1,47 +1,66 @@
...
 
-Sales channel dashboard
-=======================
+Sales Team dashboard
+====================
+
+To view your Sales Team dashboard, go to :menuselection:`CRM --> Configuration --> Sales Teams`. Any
```
Comment (meng-odoo) [inline content/applications/sales/crm/pipeline/multi_sales_team.rst:45]: Removing "you"

`To view the Sales Team dashboard, go to :menuselection:CRM --> Configuration --> Sales Teams. Any`

Please note that I removed the backticks around the menuselection items in my corrected quote.

File: content/applications/sales/crm/pipeline/multi_sales_team.rst [hunk 9]
```diff
@@ -1,47 +1,66 @@
...
+
+Each tile gives an overview of the Sales Team's open opportunities, quotations, sales orders, and
+expected revenue, as well as a bar graph of new opportunities per week and an invoicing progress
+bar.
+
+.. image:: multiple_sales_teams/sales-team-overview.png
+   :align: center
+   :alt: Sales Team Overview
+
+Clicking on the three dots in the corner of a tile opens a navigational menu that lets you quickly
```
Comment (meng-odoo) [inline content/applications/sales/crm/pipeline/multi_sales_team.rst:56]: Improving tense

`Click on the three dots in the corner of a tile to open a navigational menu that lets you quickly`

File: content/applications/sales/crm/pipeline/multi_sales_team.rst [hunk 10]
```diff
@@ -1,47 +1,66 @@
...
+view documents or reports, create new quotations or opportunities, pick a color for this team, or
+access its configuration page.
+
+.. image:: multiple_sales_teams/team-overview-three-dot-menu.png
+   :align: center
+   :alt: Three Dot Menu
+
+Clicking on the **Pipeline** button brings you directly to that team's CRM pipeline.
```
Comment (meng-odoo) [inline content/applications/sales/crm/pipeline/multi_sales_team.rst:64]: Improving tense

`Click on the **Pipeline** button to go directly to that team's CRM pipeline.`

File: content/applications/sales/crm/pipeline/multi_sales_team.rst [hunk 11]
```diff
@@ -1,47 +1,66 @@
...
-Create a new sales channel
-==========================
 
-To create a new *Sales Channel*, go to :menuselection:`Configuration --> Sales Channels`.
+Create a new Sales Team
+=======================
 
-There you can set an email alias to it. Every message sent to that email
-address will create a lead/opportunity.
+To create a new *Sales Team*, go to :menuselection:`CRM --> Configuration --> Sales Teams` then
```
Comment (meng-odoo) [inline content/applications/sales/crm/pipeline/multi_sales_team.rst:12]: I don't think this instance of "Sales Team" needs to be italicized. Let's italicize it once at the very beginning for emphasis/understanding, then leave it unitalicized but capitalized in the rest of the doc.

`To create a new Sales Team, go to :menuselection:CRM --> Configuration --> Sales Teams then`

Please note that I removed the backticks around the menuselection items in my corrected quote.

**Review by meng-odoo — CHANGES_REQUESTED** [review]
@StraubCreative Hi Zac! Thanks for your feedback on my first review--I'll keep all of that in mind when writing my future reviews as well.

I think this one makes a lot of improvements :) Please let me know if you have any more suggestions.

**Comment by StraubCreative** [comment]
@meng-odoo 
Second pass changes were great! Doc sounds a lot more confident, instructional, and easier to read imo.
Please find your change requests on latest commit [e8d2765](https://github.com/odoo/documentation/commit/e8d2765440336f589ab3e3961f58f85adfa28269) and let me know if we need anything else, otherwise we can pass over to Doc Review :)

**Review by meng-odoo — APPROVED** [review]
@StraubCreative Thanks Zac! Everything looks good to me. I'll go ahead and tag Doc Review :)

**Review by AntoineVDV — APPROVED** [review]
Perfect! :ok_hand: 

@robodoo r+

**Comment by AntoineVDV** [comment]
@meng-odoo @StraubCreative could you just squash the two commits into one so that the PR can be merged?

**Comment by StraubCreative** [comment]
> @meng-odoo @StraubCreative could you just squash the two commits into one so that the PR can be merged?

Should be good to go @AntoineVDV 🚀

**Pull request closed without merging**